spec/jobs/invoices/prepaid_credit_job_spec.rb

```diff
@@ -3,19 +3,20 @@
 require 'rails_helper'
 
 RSpec.describe Invoices::PrepaidCreditJob, type: :job do
-  let(:invoice) { create(:invoice, customer: customer) }
+  let(:invoice) { create(:invoice, customer:, organization: customer.organization) }
   let(:customer) { create(:customer) }
-  let(:subscription) { create(:subscription, customer: customer) }
-  let(:wallet) { create(:wallet, customer: customer, balance: 10.0, credits_balance: 10.0) }
+  let(:subscription) { create(:subscription, customer:) }
+  let(:wallet) { create(:wallet, customer:, balance: 10.0, credits_balance: 10.0) }
   let(:wallet_transaction) do
-    create(:wallet_transaction, wallet: wallet, amount: 15.0, credit_amount: 15.0, status: 'pending')
+    create(:wallet_transaction, wallet:, amount: 15.0, credit_amount: 15.0, status: 'pending')
   end
   let(:fee) do
-    create(:fee,
+    create(
+      :fee,
       fee_type: 'credit',
       invoiceable_type: 'WalletTransaction',
       invoiceable_id: wallet_transaction.id,
-      invoice: invoice
+      invoice:
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)